lxd/storage/drivers/driver_zfs_volumes.go

```diff
@@ -600,6 +601,13 @@ func (d *zfs) CreateVolumeFromCopy(vol Volume, srcVol Volume, copySnapshots bool
 func (d *zfs) CreateVolumeFromMigration(vol Volume, conn io.ReadWriteCloser, volTargetArgs migration.VolumeTargetArgs, preFiller *VolumeFiller, op *operations.Operation) error {
 	// Handle simple rsync and block_and_rsync through generic.
 	if volTargetArgs.MigrationType.FSType == migration.MigrationFSType_RSYNC || volTargetArgs.MigrationType.FSType == migration.MigrationFSType_BLOCK_AND_RSYNC {
+		// Round up the volume size to avoid running into "no space left on device".
```

Comment:
@monstermunchkin is this part still needed now you're reading the exact size from the source?

I'm wondering why we didn't encounter issues with this when migrating instance block volumes.